src/app/core/tools/InteractionTool.ts

```diff
@@ -5,29 +5,17 @@ import {Event} from "core/utils/Events";
 import {isPressable} from "core/utils/Pressable";
 import {LEFT_MOUSE_BUTTON} from "core/utils/Constants";
 
-import {CircuitDesigner, IOObject} from "core/models";
-
 import {DefaultTool} from "./DefaultTool";
 
 import {EventHandler} from "./EventHandler";
+import {FindPressableObjects} from "core/utils/ComponentUtils";
```

Comment:
nit: move this to the rest of `core/utils` imports above (see https://docs.opencircuits.io/Guides/Imports)